client/sites-dashboard/components/sites-grid-action.tsx

```diff
@@ -0,0 +1,66 @@
+import { Gridicon } from '@automattic/components';
+import { css } from '@emotion/css';
+
+interface SitesGridActionCTA extends React.ButtonHTMLAttributes< HTMLAnchorElement > {
+	title: string;
+	href: string;
+	children?: React.ReactNode;
+	onClick?: () => void;
+}
+
+interface SitesGridActionProps {
+	children: React.ReactNode;
+	icon?: string;
+	ctaProps?: SitesGridActionCTA | false;
+}
+
+const SITES_GRID_ACTION = css( {
+	'--color-site-grid-action-icon': '#ea303f',
+	display: 'flex',
+	alignItems: 'center',
+	position: 'absolute',
+	backgroundColor: 'var(--studio-white)',
+	padding: '16px 20px',
+	boxSizing: 'border-box',
+	borderRadius: 4,
+	left: 6,
+	top: 6,
+	width: 'calc( 100% - 12px )',
+	boxShadow: '0 0 4px var(--studio-gray-60)',
+} );
+
+const CONTENT = css( {
+	display: 'flex',
+	alignItems: 'center',
+	marginRight: 16,
+	'.gridicon': {
+		color: 'var(--color-site-grid-action-icon)',
+		marginRight: 4,
+		flexShrink: 0,
+	},
+} );
+
+const CTA = css( {
+	'--color-link': '#111619',
```

Comment:
What's the point of this CSS variable here? It's a simple `a` element, right? Is it used by any child?